### PR TITLE
Added social icons for fellows #11

### LINKED ISSUE #11: Add social icons for fellows
At the moment, it only displays the fellow name and location. To help people network, we should add social icons to each of the fellow cards.

Screenshot 2021-09-30 at 14 24 40
- [ ] Add LinkedIn icons to the card
- [ ] Add Twitter icons to the card
- [ ] Add Resume icons to the card
- [ ] Add GitHub icons to the card

### PR DESCRIPTION
**Pull request by jessielin34**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

Fixes #11

### CODE REVIEW

**Comment by Ankur6702** [comment]
Hello @akshgpt7 , just wanted to let you know that me and @jessielin34 have resolved issue 11. We created a new social icons div and made some modifications to the CSS. Currently, we haven't updated the fellows.yml file to link it to the actual social links of all the fellows. We are planning to update this file at the very end because in other issues there are significant changes in this file made by other fellows. 
Could you please review our changes? 
Thank you!

File: Gemfile.lock [hunk 1]
```diff
@@ -37,7 +37,7 @@ GEM
       rexml
     kramdown-parser-gfm (1.1.0)
       kramdown (~> 2.0)
-    liquid (4.0.3)
+    liquid (4.0.4)
```
Comment (akshgpt7) [inline Gemfile.lock]: You don't need to commit the Gemfile.lock. Can you omit this file from the PR please?

File: bin/bundle [hunk 2]
```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
```
Comment (akshgpt7) [inline bin/bundle]: Don't need any of the files in the `bin/` directory. Please remove these files too.

**Review by akshgpt7 — CHANGES_REQUESTED** [review]
Thanks for working on this. The icons look awesome!
Can you just:
1. Fix the merge conflict that has come up
2. Remove the files that are not needed (see my review comments)

It'll be ready to merge it after that

**Comment by Ankur6702** [comment]
Hey @akshgpt7 , thanks for reviewing our PR. We have updated the PR as per the suggestions given by you. You can look into it now. Let me know if any change is still required from our end!
Thanks

**Comment by akshgpt7** [comment]
Hey, can you fix the merge conflicts, will be good to merge then!

**Comment by akshgpt7** [comment]
Also mention the issue number in the PR description that it fixes.
"Fixes #x"

**Comment by Ankur6702** [comment]
Fixes #11 
@akshgpt7 I've resolved the merge conflicts. Please let me know if its ready to be merged or not!

**Comment by akshgpt7** [comment]
You need to add "Fixes #11" to the PR description, not as a comment.

**Comment by akshgpt7** [comment]
LGTM, Thanks for working on this!

**Pull request merged**